quickstarts/microsoft.network/aad-domainservices/metadata.json

```diff
@@ -5,6 +5,6 @@
   "description": "This template deploys an Managed Azure Active Directory Domain Service with required VNet and NSG configurations.",
   "summary": "Deploy Managed AADDS with required VNet and NSG configurations",
   "githubUsername": "rahulkhengare",
-  "dateUpdated": "2021-05-14",
+  "dateUpdated": "2021-06-11",
   "validationType": "Manual"
```

Comment:
Can you remove this and see if automated testing passes?  I can't see the reason why it's needed...

If it does work, we should update the readme to remove the antiquated PS commands

Reply:
@bmoore-msft Can you look at the deployment failure?  thx

Reply:
reverted